config/cloud.cfg.tmpl

```diff
@@ -305,6 +320,11 @@ system_info:
 {% elif variant == "openmandriva" %}
   network:
     renderers: ['network-manager', 'networkd']
+{% elif variant == "raspberry-pi-os" %}
+  network:
+    dhcp_client_priority: [dhclient]
+    renderers: ['netplan', 'network-manager', 'networkd', 'eni']
```

Comment:
Which of these does raspberry-pi-os support and plan to use?

Reply:
> Which of these does raspberry-pi-os support and plan to use?

In the updated PR, this has been reduced to netplan & network-manager, both of which we want to offer.